src/components/Profile/TabItem.tsx

```diff
@@ -10,10 +10,20 @@ interface TabProps extends ButtonHTMLAttributes<HTMLButtonElement> {
 }
 
 const TabItem = ({ item, value, isActive, onClick }: TabProps) => {
+  const [name, setName] = useState<string>(item);
+
+  useEffect(() => {
+    if (name === 'posts') {
+      setName('ground');
+    } else if (name === 'followers') {
+      setName('follower');
+    }
+  }, [name]);
```

Comment:
useEffect로 감싸는게 맞는건지 의문이 들어요 .. 의견 주시면 감사하겠습니다 😅

Reply:
```suggestion
  const [name, setName] = useState<string>('');

  useEffect(() => {
    if (item === 'posts') {
      setName('ground');
    } else if (item === 'followers') {
      setName('follower');
    }
  }, [item]);
```

props 로 전달된 item에 따라 name 이 바뀌어야 한다면, 위 처럼 작성하는게 코드의 흐름을 조금 더 파악하기 쉬울 것 같습니다. 🤔

Reply:
이 경우에는 useState와 useEffect를 쓰지 않아도 될 것 같습니다
```ts
const name = item === 'posts' ? 'ground' : (item === 'followers' ? 'follower' : 'following');
```
그냥 이렇게 쓰면 됩니다~ㅎㅎ